Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.yml

```diff
@@ -1405,3 +1406,5 @@ script:
 defaultmapperin: QRadar-mapper
 defaultclassifier: QRadar
 fromversion: 6.0.0
+tests:
+- No tests (auto formatted)
```

Comment:
It does have a test, though it should still be automatically picked up - in any case, better add it here for clarity.